lib/src/types.ts

```diff
@@ -15,8 +15,8 @@ export type RepoMeta = {
   name: string,
   description: string,
   projectType: ProjectType,
-  homepage: string,
-  repository: string
+  homepage?: string,
```

Comment:
This causes the frontend to break here since the types are now string | undefined:

https://github.com/valist-io/valist/blob/df3564e7e2462c6de6cf41c0b08297d3f879ba08/relay/components/Projects/EditProjectMetaForm.tsx#L30

Reply:
I added some small checks to the line:
if (props.meta.homepage) setHomepage(props.meta.homepage);
if (props.meta.repository) setRepository(props.meta.repository);